### DISCUSSION TITLE
Tidak dapat kirim Link

### DISCUSSION THREAD

**Discussion by mu-hasan** [kirimwa-id/forum#185] [Tanya Jawab]
Halo, saya coba kirim pesan menggunakan Link di dalamnya tetapi yang terkirim malah berupa plain text. Apakah fitur ini sudah tidak didukung oleh WA? Terima kasih

<img width="544" alt="Screenshot 2024-02-15 at 12 10 16" src="https://github.com/kirimwa-id/forum/assets/8762248/40dfa9dc-5fd5-435b-8c3b-01452a795c26">

**Comment by underworld14** [comment]
Izin up

**Comment by kotak47** [comment]
ini biasanya nomor wa tidak tersimpan sebagai kontak di nomor penerima pesan
